rasa/core/tracker_store.py

```diff
@@ -451,12 +489,17 @@ def save(self, tracker: DialogueStateTracker) -> None:
 
         self.db.put_item(Item=serialized)
 
-    def serialise_tracker(self, tracker: "DialogueStateTracker") -> Dict:
-        """Serializes the tracker, returns object with decimal types."""
-        d = tracker.as_dialogue().as_dict()
-        d.update({"sender_id": tracker.sender_id})
-        # DynamoDB cannot store `float`s, so we'll convert them to `Decimal`s
-        return core_utils.replace_floats_with_decimals(d)
+    def serialise_tracker(
+        self,
+        tracker: "DialogueStateTracker",
+    ) -> Dict:
+        """Serializes the tracker, returns object with decimal types.
+
+        DynamoDB cannot store `float`s, so we'll convert them to `Decimal`s.
+        """
+        return core_utils.replace_floats_with_decimals(
+            super().serialise_tracker(tracker)
```

Comment:
oh in that case, let's keep it as a static method because, and just use `TrackerStore.serialise_tracker()`. Unless we have a good reason to change it from a static method (other than making mypy happy)

Reply:
Done: https://github.com/RasaHQ/rasa/pull/10965/commits/f0103e1b6a9b4828a6fe1f78eefd35b127eaa087